app/components/progressive_image/progressive_image.js

```diff
@@ -189,4 +189,10 @@ const styles = StyleSheet.create({
         alignItems: 'center',
         justifyContent: 'center',
     },
+    attachmentMargin: {
+        marginTop: 2.5,
```

Comment:
Why 2.5, where are these values coming from? Why not 5 as it was?

Reply:
I believe it's because of the absolutefill being used, it was pushing out the image 2.5 more from the left/top. I tried other methods without the absolutefill, but this was the most accurate.